environment-linux-aarch64.yml

```diff
@@ -25,21 +26,19 @@ dependencies:
   - einops=0.3.0
   - kornia=0.6
   - torchmetrics=0.7.0
-  - transformers=4.21.3
+  - transformers=4.23
   - torch-fidelity=0.3.0
   - tokenizers>=0.11.1,!=0.11.3,<0.13
   - pip:
+      - dependency_injector==4.40.0
```

Comment:
No longer necessary - same for all platforms

Reply:
Since when? Earlier today the `python scripts/preload_models.py --no-interactive` did not work without it and when I look at it now, the import is still in it ...

Reply:
Hmmmm, it showed me the comment next to `getpass_asterisk`, which the last comment is also meaning, now I'm confused :-o